frontend/src/pages/VendorDashboard.jsx

```
@@ -0,0 +1,371 @@
+                    </Link>
+                    <Link 
+                      to={`/edit-property/${property._id}`}
+                      className="bg-yellow-50 text-yellow-600 py-2 px-4 rounded hover:bg-yellow-100 transition duration-300"
+                    >
+                      <Edit size={16} className="inline mr-1" /> Edit
+                    </Link>
+                    <button 
+                      onClick={() => handleDeleteProperty(property._id)}
+                      className="bg-red-50 text-red-600 py-2 px-4 rounded hover:bg-red-100 transition duration-300 cursor-pointer"
+                    >
+                      <Trash size={16} className="inline mr-1" /> Delete
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)